shared/fs/common/index.stories.js

```diff
@@ -64,6 +65,14 @@ export const commonProvider = {
     enableDriver: Sb.action('enableDriver'),
     openInSystemFileManager: Sb.action('openInSystemFileManager'),
   }),
+  FolderViewFilter: ({path}: {path: Types.Path}) => ({
+    filter: folderViewFilter,
+    onUpdate: (filter: string) => {
+      Sb.action('onUpdate')(filter)
+      folderViewFilter = filter
```

Comment:
ooh, this is cool

Reply:
Doesn't work unfortunately; I forgot to tested after adding it. Will remove to avoid confusion ...

Reply:
I guess this makes sense. This "connector" doesn't get re-called since the store never changes